test/torchaudio_unittest/rnnt/utils.py

```diff
@@ -371,30 +381,44 @@ def get_numpy_data_B2_T4_U3_D3(dtype=np.float32):
     return data, ref_costs, ref_gradients
 
 
-def get_numpy_random_data(
-    max_B=8, max_T=128, max_U=32, max_D=40, blank=-1, dtype=np.float32, seed=None
+def get_random_data(
+    max_B=8,
+    max_T=128,
+    max_U=32,
+    max_D=40,
+    blank=-1,
+    dtype=torch.float32,
+    device=torch.device("cpu"),
+    seed=None,
 ):
     if seed is not None:
-        np.random.seed(seed=seed)
+        torch.manual_seed(seed=seed)
 
     if blank != -1:
         raise ValueError("blank != -1 is not supported yet.")
 
-    B = np.random.randint(low=1, high=max_B)
-    T = np.random.randint(low=5, high=max_T)
-    U = np.random.randint(low=5, high=max_U)
-    D = np.random.randint(low=2, high=max_D)
-
-    logit_lengths = np.random.randint(low=5, high=T + 1, size=(B,), dtype=np.int32)
-    target_lengths = np.random.randint(low=5, high=U + 1, size=(B,), dtype=np.int32)
-    max_src_length = np.max(logit_lengths)
-    max_tgt_length = np.max(target_lengths)
-    targets = np.random.randint(
-        low=0, high=D - 1, size=(B, max_tgt_length), dtype=np.int32
+    B = random.randint(1, max_B - 1)
+    T = random.randint(5, max_T - 1)
+    U = random.randint(5, max_U - 1)
+    D = random.randint(2, max_D - 1)
```

Comment:
did you set the seed for [random package](https://docs.python.org/3/library/random.html#random.seed)?